PigLatin/PigLatin.cs

```diff
@@ -1,21 +1,62 @@
 ﻿using System;
+using System.Collections.Generic;
 
 namespace PigLatin
 {
     class Program
     {
-        public static void Main()
+        public static void Main(string[] args)
         {
-            // your code goes here
-
-            // leave this command at the end so your program does not close automatically
-            Console.ReadLine();
+        //stores user input into string sentence    
+        Console.WriteLine("Enter a sentence to convert to PigLatin:");
+        string sentence = Console.ReadLine();
+        //takes translated sentence from ToPigLatin and then writes it into console
+        string pigLatin = ToPigLatin(sentence);
+        Console.WriteLine(pigLatin.ToLower());
         }
-        
-        public static string TranslateWord(string word)
+
+        //translates sentence
+        static string ToPigLatin (string sentence)
         {
-            // your code goes here
-            return word;
+            //creates list of words that the translated words are added to                        
+            List<string> latin = new List<string>();
+
+            //splits user entered input
+            foreach (string word in sentence.Split(' '))
+            {
+                //checks position each vowel and y
+                int firstVowelIndex = word.IndexOfAny(new char[] {'A','E','I','O','U','a','e','i','o','u',});
+                int firstYIndex = word.IndexOfAny(new char[] {'Y','y'});
+                
+                //checks if Y acts as vowel
+                if ((firstYIndex > 0) && (firstYIndex < firstVowelIndex) && (word.ToLower().Contains("y")))
```

Comment:
if firstYIndex is greater than 0 then the word must contain a 'y' so that last condition is not really needed.